src/js/getNextJokeIdx.js

```diff
@@ -0,0 +1,7 @@
+export const getNextJokeIdx = (currentJokeIdx, upperBound) => {
```

Comment:
This looks great! Another way if you're interested is using a conditional statement like this: 

`const getNextJokeIdx = (currentJokeIdx, upperBound) => (
  currentJokeIdx + 1 > upperBound ? 0 : currentJokeIdx + 1
);`

No need to change this though, just sharing!